src/reducers/controls.ts

```diff
@@ -36,11 +40,24 @@ export interface MeasurementsControlState {
 
 export interface ControlsState extends BasicControlsState, MeasurementsControlState {}
 
+interface Defaults {
+  distanceMeasure: string
+  layout: Layout
+  geoResolution: string
+  filters: Record<string, any>
+  filtersInFooter: string[]
+  colorBy: string
+  selectedBranchLabel: string
+  tipLabelKey: typeof strainSymbol
+  showTransmissionLines: boolean
+  sidebarOpen?: boolean
+}
```

Comment:
I'd put this up around L15, so it's defined before being used in the (current) L18 -- was briefly confused where `Defaults` was even coming from.

Reply:
Done: https://github.com/nextstrain/auspice/blob/46588036488a0132423f7eb01815aa658c1686e3/src/reducers/controls.ts#L15-L32